ckanext/who_afro/helpers.py

```diff
@@ -200,4 +200,3 @@ def get_license(license_id):
             return license
     else:
         return {}
-
```

Comment:
This is me being mega picky but there is no need to delete this line.

Reply:
That's weird, that line is present for me locally and my local branch is synced with the remote. I'm adding another line locally then push.